packages/arco-lib/src/components/Form/FormControl.tsx

```diff
@@ -0,0 +1,95 @@
+import { Form } from '@arco-design/web-react';
+import { ComponentImpl, implementRuntimeComponent, Text } from '@sunmao-ui/runtime';
+import { css, cx } from '@emotion/css';
+import { Type, Static } from '@sinclair/typebox';
+import { FALLBACK_METADATA, getComponentProps } from '../../sunmao-helper';
+import { FormControlPropsSchema as BaseFormControlPropsSchema } from '../../generated/types/Form';
+import { DragComponentTips } from './DragComponentTips';
+import { FormControlErrorMessage } from './FormControlErrorMessage';
+
+const FormControlPropsSchema = Type.Object(BaseFormControlPropsSchema);
+const BaseFormControl = Form.Item;
+
+const FormControlImpl: ComponentImpl<Static<typeof FormControlPropsSchema>> = props => {
+  const { label, errorMsg, ...cProps } = getComponentProps(props);
+  const { elementRef, slotsElements, customStyle } = props;
+
+  if (!cProps.colon) {
+    Reflect.deleteProperty(cProps, 'colon');
+  }
+  if (cProps.labelAlign === 'unset') {
+    Reflect.deleteProperty(cProps, 'labelAlign');
+  }
+
+  const formControlProps = {
+    className: cx(
+      'sunmao-form-control-layout',
+      css`
+        ${customStyle?.content}
+        svg {
+          display: inherit;
+        }
+        & * {
+          word-wrap: normal;
+        }
+        & label {
+          white-space: inherit !important;
+        }
+      `
+    ),
+  };
+
+  return (
+    <BaseFormControl
+      label={<Text cssStyle="display:inline-block" value={label} />}
+      ref={elementRef}
+      {...formControlProps}
+      {...cProps}
+      labelAlign={cProps.labelAlign as any}
```

Comment:
Why as any?

Reply:
Both `Form Stack` and `Form Control` can set this property, but when Form Control has a default value('right' or 'left'), the one on Form Stack will not work, so you can't set labelAlign uniformly for all Formcontrol
So I added an 'unset' that arco didn't have as a default value for `labelAlign`